src/components/Settings/UserSearchInput.js

```diff
@@ -0,0 +1,72 @@
+import React, {useEffect} from "react";
+import {useDebounce} from "use-debounce";
+import inatjs from "inaturalistjs";
+import {Image, Text, TextInput, View} from "react-native";
+import {textStyles, viewStyles} from "../../styles/settings/settings";
+import Pressable from "react-native/Libraries/Components/Pressable/Pressable";
+
+const UserSearchInput = ( { onUserChanged} ): React.Node => {
+  const [hideResults, setHideResults] = React.useState( true );
+  const [userResults, setUserResults] = React.useState( [] );
+  const [userSearch, setUserSearch] = React.useState( "" );
+  // So we'll start searching only once the user finished typing
+  const [finalUserSearch] = useDebounce( userSearch, 500 );
+
+  useEffect( () => {
```

Comment:
Let's reuse `sharedHooks/useRemoteSearchResults.js` here

Reply:
Fixed in latest commit